src/organelles/dots.yml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Should all the `.yml` files be cell-zoom space folder?

Reply:
Good thinking, just moved everything